.gitignore

```diff
@@ -13,3 +13,4 @@ Thumbs.db
 *.user
 
 /node_modules/**
+.idea
```

Comment:
There is no line break at the end of the file :D

Reply:
I agree!

Reply:
Yeah, critical issue

Reply:
co słychać chłopaki? :D